### PR TITLE
[SMP-1343] SMP 0.5.5 patch release notes

### PR DESCRIPTION
**Pull request by brian-f-harness**
SMP 0.5.5 patch release notes

For reviewers, [a preview](https://64764fc92bbffa0509a57362--harness-developer.netlify.app/release-notes/self-managed-enterprise-edition) is available.

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/self-managed-enterprise-edition.md [hunk 1]
```diff
@@ -14,7 +14,60 @@ import delete_project from './static/delete-project.png'
...
+## Latest - May 31, 2023, patch release for version 79306
+
+Patch releases for Harness Self-Managed Enterprise Edition include minor bug fixes and updates to address potential security vulnerabilities.
+
+This release includes the following Harness module and component versions.
+
+| **Name** | **Version** |
+| :-- | :--: |
+| Helm Chart | [0.5.5](https://github.com/harness/helm-charts/releases/tag/harness-0.5.5) |
+| NG Manager | 79306 |
```
Comment (akshay-khole) [inline release-notes/self-managed-enterprise-edition.md:26]: this version is incorrect, please reverify all versions

**Review by akshay-khole — CHANGES_REQUESTED** [review]

File: release-notes/self-managed-enterprise-edition.md (hunk 1 above)
Comment (brian-f-harness) [reply release-notes/self-managed-enterprise-edition.md]: @akshay-khole updated NG Manager to 78929. Reverified all other image versions. Thank you.

File: release-notes/self-managed-enterprise-edition.md [hunk 2]
```diff
@@ -14,7 +14,60 @@ import delete_project from './static/delete-project.png'
...
 
-## Latest - May 12, 2023, patch release for version 78926
+## Latest - May 31, 2023, patch release for version 79306
+
+Patch releases for Harness Self-Managed Enterprise Edition include minor bug fixes and updates to address potential security vulnerabilities.
+
+This release includes the following Harness module and component versions.
+
+| **Name** | **Version** |
+| :-- | :--: |
```
Comment (akshay-khole) [inline release-notes/self-managed-enterprise-edition.md:24]: note for future release-  is this section needed, helm-charts release has list of images, copying in multiple places can cause errors  , lets ask Prateek

**Review by akshay-khole — DISMISSED** [review]

**Review by pratmit — DISMISSED** [review]

**Review by douglas-j-bothwell — APPROVED** [review]

File: release-notes/self-managed-enterprise-edition.md [hunk 3]
```diff
@@ -14,7 +14,60 @@ import delete_project from './static/delete-project.png'
 ```
 Review the notes below for details about recent changes to Harness Self-Managed Enterprise Edition, NextGen. For release notes for FirstGen Self-Managed Enterprise Edition, go to [Self-Managed Enterprise Edition release notes (FirstGen)](/docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes).
 
-## Latest - May 12, 2023, patch release for version 78926
+## Latest - May 31, 2023, patch release for version 79306
```
[the remark this answers is not in the document]
Comment (brian-f-harness) [reply release-notes/self-managed-enterprise-edition.md:17]: Nice catch, @douglas-j-bothwell. I'll update. It went out today.

**Pull request merged**